config/core/test-configs.yaml

```diff
@@ -1250,6 +1250,8 @@ device_types:
     mach: x86
     arch: x86_64
     boot_method: grub
+    filters:
+      - blocklist: {lab: ['lab-collabora']}
```

Comment:
Confirmed on staging that this board is not running any tests any more in lab-collabora:
https://staging.kernelci.org/test/job/next/branch/master/kernel/next-20220527/